src/flags/app.js

```
@@ -5,6 +5,19 @@ import type {Flag} from 'cli-engine-command/lib/flags'
 import {merge} from '.'
 import Git from '../git'
 
+function multipleRemotesError (gitRemotes) {
+  return `Multiple apps in git remotes
```

I would return the error here, `return new Error('...`